Doctrine/Phpcr/Form/Type/StaticContentType.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr\Form\Type;
+
+use Doctrine\Bundle\PHPCRBundle\Form\DataTransformer\DocumentToPathTransformer;
+use Doctrine\ODM\PHPCR\DocumentManager;
+use Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr\StaticContent;
+use Symfony\Cmf\Bundle\ContentBundle\Form\Type\StaticContentType as ModelStaticContentType;
+use Symfony\Cmf\Bundle\TreeBrowserBundle\Form\Type\TreeSelectType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class StaticContentType extends AbstractType
+{
+    /**
+     * @var DocumentManager
+     */
+    private $documentManager;
+
+    /**
+     * @var string
+     */
+    private $contentBasePath;
+
+    /**
+     * @param DocumentManager $documentManager
+     * @param string          $contentBasePath
+     */
+    public function __construct(DocumentManager $documentManager, $contentBasePath)
+    {
+        $this->documentManager = $documentManager;
+        $this->contentBasePath = $contentBasePath;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
```

Comment:
maybe similar to this:
https://github.com/FriendsOfSymfony/FOSHttpCache/blob/master/src/ProxyClient/Varnish.php#L25-L27

note that the "additional" comes because in that example there is a base
class and we add to it. i feel `-` is better to indicate the list than
`*` as `*` is already used to denote the comment section.